src/components/select/single-item-mode.js

```diff
@@ -6,46 +6,46 @@
  * @author BBC Children's D+E
  * @license Apache-2.0
  */
-import * as a11y from "../../core/accessibility/accessibility-layer.js";
-
-const currentCellIsLast = scene => scene.grid.getCurrentPageKey() === scene._cells[scene._cells.length - 1].button.key;
+const removeEvents = cell => {
+    cell.button.off("pointerover", cell.over);
+    cell.button.off("pointerout", cell.out);
+};
 
 export const create = scene => {
     const continueButton = scene.layout.buttons.continue;
     if (!continueButton) return false;
 
-    a11y.removeButton(continueButton);
-    a11y.removeButton(scene.layout.buttons.next);
-    a11y.removeButton(scene.layout.buttons.previous);
-    a11y.reset();
+    const onBlur = () => scene.grid.showPage(0);
+    window.addEventListener("blur", onBlur);
```

Comment:
This is causing undesired behaviour when switching browser tab -> switching back.
It looks like it jumps to the first cell then animates to the second without showing the asset.

Could it be related to the addition to the return in `showPage` ?
https://github.com/bbc/genie/pull/176/files#diff-b5d608f4ffc03234f3a1a5250497a26eR87

Reply:
can reproduce, will try and find out what it is now